cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ArtifactHttpHandlerInternal.java

```diff
@@ -90,6 +106,50 @@ public void listArtifacts(HttpRequest request, HttpResponder responder,
     }
   }
 
+  @POST
```

Comment:
Why it is a POST call? Shouldn't it be GET?

Reply:
Yes, I switched it to POST to fix an error I was seeing but I changed it back to GET now and everything works